File: cloudbot/docs/legacy/commercial-director/assistant/agents/larisa-ivanovna/commands/plan_day.ts
import { LARISA_COMMAND_ALIASES, larisaIvanovnaConfig } from "../config";
import type { GetDayBriefInput } from "../schemas/brief.schema";
import { normalizeDayBriefRequest } from "./get_day_brief";
import { runPlanDayWorkflow } from "../workflows/plan_day.workflow";
import type { DailyBriefWorkflowDeps } from "../workflows/daily_brief.workflow";

export function createPlanDayCommand(deps: DailyBriefWorkflowDeps) {
  return {
    name: "plan_day",
    aliases: [
      ...LARISA_COMMAND_ALIASES.planDay,
      ...larisaIvanovnaConfig.legacyCommandAliases.planDay,
    ],
    async execute(input: GetDayBriefInput = {}) {
      const result = await runPlanDayWorkflow(normalizeDayBriefRequest(input), deps);

      return {
        text: result.text,
        payload: result,
      };
    },
  };
}

File: cloudbot/apps/larisa_ivanovna/commands/plan_day.py
"""Команда планирования дня."""

from __future__ import annotations

from ..config import COMMAND_ALIASES
from ..schemas.brief import DayBriefRequest
from ..workflows.daily_brief import DailyBriefWorkflowDeps
from ..workflows.plan_day import run_plan_day_workflow


def build_command(deps: DailyBriefWorkflowDeps) -> dict[str, object]:
    def handler(payload: DayBriefRequest, context: dict[str, object] | None = None) -> dict[str, object]:
        result = run_plan_day_workflow(payload, deps)
        return {
            "text": str(result.get("text") or ""),
            "payload": result,
        }

    return {
        "name": "plan_day",
        "aliases": COMMAND_ALIASES["plan_day"],
        "handler": handler,
    }
